services/value-store/projects/src/events/onObjectCreated.js

```diff
@@ -7,6 +7,8 @@ import split2 from 'split2';
 
 import deleteProjects from '../lib/deleteProjects';
 import SaveStream from '../lib/SaveStream';
+// elasticsearch mapping https://goo.gl/MEGiA6
```

Comment:
Please don't use Google's URL shortener... please...

Reply:
I didn't fine one for the EC and links are sometimes too long and ugly